src/plugins/intel_cpu/tests/unit/nodes/kernels/stack_allocator.cpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <utility>
+#include <gtest/gtest.h>
+#include <cpu/x64/jit_generator.hpp>
+#include <nodes/kernels/stack_allocator.hpp>
+
+#include <cpu_memory.h>
+
+using namespace ov::intel_cpu;
+using namespace InferenceEngine;
+
+class StackAllocatorTest : public ::testing::Test, public x64::jit_generator {
+protected:
+    DECLARE_CPU_JIT_AUX_FUNCTIONS(StackAllocatorTest)
+
+    void SetUp() override {
+    }
+
+    void TearDown() override {
+    }
+
+    template<typename F>
+    F create_kernel() {
+        const status_t code = jit_generator::create_kernel();
+        if (code != dnnl::impl::status::success) {
+            IE_THROW() << "Could not create kernel. Error code: " << std::to_string(code) << ". "
+                       << "Xbyak error code: " << Xbyak::ConvertErrorToString(Xbyak::GetError());
+        }
+        return reinterpret_cast<F>(jit_ker());
+    }
+
+    void generate() override {
+        this->preamble();
+        stack_allocator_ = std::make_shared<StackAllocator>(*this);
+        kernel_();
+        stack_allocator_->commit();
```

Comment:
It will be good to have a special method `finalize` here to check if all allocations were released. Currently, if some allocations were not released at this point, the program will crash.

Reply:
`StackAllocator` has already `release` method that perform free of all allocated chunks